### ISSUE TITLE
Deleted queues' pages are still available until server restart

### ISSUE THREAD

**Issue by Shadow53** [Shadow53/student-queue#10]
If I add a queue "test" and then delete it, I am still able to interact with it and load pages, but the table has been deleted along with its entry in the 'queues' table. Restarting the server fixes this as the Socket.IO room for it is never created.

Use this for removing a namespace: http://stackoverflow.com/questions/26400595/socket-io-how-do-i-remove-a-namespace

**Comment by Shadow53** [comment]
Turns out the issue is not with Socket.IO, but Express is still serving the static files

**Comment by Shadow53** [comment]
Found [this](http://stackoverflow.com/questions/10378690/remove-route-mappings-in-nodejs-express). It involves messing with Express internals, judging from the _ in front of the property name, but it works. I doubt the authors intended on dynamically creating/removing routes, at least in the current release, so this makes sense. Still, I don't want to mislead any users by having the right static pages served and nothing working.

**Comment by Shadow53** [comment]
Turns out the link was right, but not. I cannot work with named functions because of the loop and the `path` turned out to be undefined most of the time instead of `"/admin"` or whatever else. So I decided to match on the stringified regular expression - woo!

I made some assumptions with the check to make it more specific - namely, I attempt to recreate the regular expression and check against that. It should work since I have limited queue names to alphanumeric characters. It works in testing, anyhow.

I will be working on adding a "home" page to the program so that it does not leave people with "Cannot get /", then I will be pushing my changes and making a new release.
